__TESTS/perftests/rf_functions/test_findpotwells.py

```diff
@@ -0,0 +1,175 @@
+# coding: utf8
+# Copyright 2020 CERN. This software is distributed under the
+# terms of the GNU General Public Licence version 3 (GPL Version 3),
+# copied verbatim in the file LICENCE.md.
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+# Project website: http://blond.web.cern.ch/
+
+"""
+Performance tests for the blond_common.rf_functions.potential module
+
+:Authors: **Alexandre Lasheen**
+
+"""
+
+# General imports
+# ---------------
+import sys
+import numpy as np
+import os
+import time
+
+this_directory = os.path.dirname(os.path.realpath(__file__)) + "/"
+
+# BLonD_Common imports
+# --------------------
+if os.path.abspath(this_directory + '../../../../') not in sys.path:
+    sys.path.insert(0, os.path.abspath(this_directory + '../../../../'))
+
+from blond_common.rf_functions.potential import find_potential_wells_cubic
+
+# Input data folder
+input_folder = this_directory+'/../../input/rf_functions/'
+
+
+class TestFindPotWells(object):
+
+    # Initialization ----------------------------------------------------------
+
+    def __init__(self, iterations=100):
+        
+        self.iterations = iterations
+        
+        # Defining a ramp with a program time vs. energy (warning: initial energy cannot be 0)
+    
+        from blond_common.interfaces.beam.beam import Particle
+        from blond_common.interfaces.input_parameters.ring import Ring
+        from scipy.constants import u, c, e
```

Comment:
Why here?

Reply:
Legacy ipynb copy past, moved on top

NB: the test is not working now and need to be fixed with Ring/RFStation clean up